app/controllers/users_controller.rb

```diff
@@ -13,16 +13,13 @@ def show
     @unfinished_works = Work.unfinished_works(@user)
     @completed_works = Work.completed_works(@user)
     @withdrawn_works = Work.withdrawn_works(@user)
-    render "show"
   end
 
   # GET /users/1/edit
   def edit
-    if can_edit?
-      render "edit"
-    else
+    unless can_edit?
       Rails.logger.warn("Unauthorized to edit user #{@user.id} (current user: #{current_user.id})")
-      redirect_to user_path(@user)
+      redirect_to user_path(@user) and return
```

Comment:
Is the and return needed? I have never seen that syntax before.  Is the and a short cut for a semicolon?
If so I would prefer the return on the next line, so that it is not hidden.

Reply:
will do. I think the motivation where I've seen it used is to make sure the return immediately follows the redirect, to make sure there isn't a double render, but moving it to its own line would be fine.